app/util/data_preparation/crowd_prepare_data.py

```diff
@@ -0,0 +1,96 @@
+import urllib3
+import random
+import string
+
+from util.conf import CROWD_SETTINGS
+from util.api.crowd_clients import CrowdRestClient
+from util.project_paths import CROWD_USERS
+
+
+urllib3.disable_warnings(urllib3.exceptions.InsecureRequestWarning)
+
+
+USERS = "users"
+DEFAULT_USER_PASSWORD = 'password'
+DEFAULT_USER_PREFIX = 'performance'
```

Comment:
do we need _ here? e.g. 'performance_'.
Also, need to consider renaming to 'performance_crowd_' to avoid possible confusion with jira performance users.